src/vmdeps.txt

```diff
@@ -14,3 +14,6 @@ dhcp-client bind-export-libs iproute
 
 # SELinux
 selinux-policy selinux-policy-targeted policycoreutils
+
+# coreos-assembler
+python3 python3-gobject-base buildah podman skopeo iptables iptables-libs
```

Comment:
this doesn't look EL7 friendly - are we abandoning that endeavor ?

Reply:
TBH, I had been focused on just getting this to work in F29

We already know that the version of `supermin` in EL7 doesn't support the necessary flags to setup the appliance, so it might be time to call it abandoned.

Reply:
yeah - just in case let's add `#FEDORA ` to the front of the line?

Reply:
FIxed in last push